src/components/ContributorProjectsCollab.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { useQuery } from '@apollo/client'
+import {
+    Box,
+    Card,
+    Grid,
+    Typography
+} from '@material-ui/core'
+
+import LoadingProgress from './LoadingProgress'
+import ProjectTile from './ProjectTile'
+import { GET_CONTRIBUTOR_PROJECTS } from '../operations/queries/ContributorQueries'
+
+const ContributorProjectsCollab = (props) => {
```

Comment:
Not sure about this component name (remember to [avoid abbreviations](https://github.com/setlife-network/trinary/blob/develop/CONTRIBUTING.md#3-avoid-abbreviations))

Suggested change to `ContributorProjectInvolvement`? Or just `ContributorInvolvement`?